axolotl-web/src/components/AddGroupMembersModal.vue

```diff
@@ -82,7 +98,7 @@ export default {
     }
   },
   mounted(){
-    this.contacts = this.$store.state.contacts
+    this.contacts = this.$store.state.contacts.filter(c=>c.UUID[0]!=0||c.UUID[c.UUID.length-1]!=0)
```

Comment:
This condition is at several places, could you please add it as a method "isContactXXX" returning a boolean on Contact?